### PR TITLE
Launch DirectDebit as a payment option for the UK

### PR DESCRIPTION
**Pull request by rupertbates**
## Why are you doing this?
After the successful test of Direct Debit as a payment method we now have sign off to push it live.

[**Trello Card**](https://trello.com/c/KzR37rCl/1458-put-direct-debit-live)

## Screenshots
screen shot 2018-04-25 at 15 29 48

### CODE REVIEW

File: assets/pages/regular-contributions/components/regularContributionsPayment.jsx [hunk 1]
```diff
@@ -77,9 +77,8 @@ function getStatusMessage(
  */
 function RegularContributionsPayment(props: PropTypes, context) {
   let directDebitButton = null;
-  const isDirectDebitEnabled = props.abParticipations.directDebitTest === 'directDebit' || (window.guardian && window.guardian.directDebitEnable);
 
-  if (props.country === 'GB' && isDirectDebitEnabled) {
+  if (props.country === 'GB') {
```
Comment (svillafe) [inline assets/pages/regular-contributions/components/regularContributionsPayment.jsx]: Is DD only for the UK (GB) or also for GBP Transaction countries like Gibraltar, etc?

Comment (rupertbates) [reply assets/pages/regular-contributions/components/regularContributionsPayment.jsx]: It's for anyone with a UK bank account really

Comment (svillafe) [reply assets/pages/regular-contributions/components/regularContributionsPayment.jsx]: Cool, in that case, I think the check as it current stands makes sense.

**Comment by rupertbates** [comment]
Good question @svillafe I've put a card in to check: https://trello.com/c/2xZqbbkg/1459-check-that-mma-for-cancelling-contributions-works-with-direct-debit

**Pull request merged**